runtime/Makefile

```diff
@@ -39,6 +42,17 @@ ifneq ($(CHPL_MAKE_LAUNCHER),none)
 	@$(MAKE) -f Makefile.help MAKE_LAUNCHER=1 CHPL_MAKE_RULE=$* $*
 endif
 
+$(CHPL_ENV_HEADER):
+	echo $(RUNTIME_INCLS)
+	@echo $(CHPL_ENV_HEADER)
```

Comment:
Not sure if this will fix it, but I think you need to `@mkdir -p $(CHPL_ENV_HEADER_DIR)` before trying to create CHPL_ENV_HEADER